devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -204,7 +204,8 @@ public void execute(Task test) {
                     // create a custom configuration for devmode
                     configurations.create(DEV_MODE_CONFIGURATION_NAME).extendsFrom(
                             configurations.getByName(JavaPlugin.COMPILE_ONLY_CONFIGURATION_NAME),
-                            configurations.getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME));
+                            configurations.getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME),
+                            configurations.getByName(JavaPlugin.RUNTIME_ONLY_CONFIGURATION_NAME));
```

Comment:
Shouldn't RUNTIM_CLASSPATH include RUNTIME_ONLY?

Reply:
Right, this works, I pushed the update.